src/nvim/option.c

```diff
@@ -7189,7 +7189,9 @@ Dictionary get_vimoption(String name, Error *err)
 Dictionary get_all_vimoptions(void)
 {
   Dictionary retval = ARRAY_DICT_INIT;
-  for (size_t i = 0; i < PARAM_COUNT; i++) {
+  for (size_t i = 0; i < PARAM_COUNT - 1; i++) {
```

Comment:
@bfredl this ones on you LUL

Reply:
use `options[i].fullname` directly instead of magic -1 + assert.